compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -182,6 +182,9 @@ int entry(int argc, char **argv)
       std::cerr << "ERROR: Optimized graph is invalid" << std::endl;
       return 255;
     }
+
+    // quantize the graph
+    optimizer.quantize(graph);
```

Comment:
let's move this before `validate`. `validate` will do some minimum graph validation.